src/server/pkg/testutil/auth.go

```diff
@@ -155,6 +156,7 @@ func activateAuth(tb testing.TB) {
 	}
 	tokenMap[AdminUser] = resp.PachToken
 	ioutil.WriteFile(adminTokenFile, []byte(resp.PachToken), 0644)
+	config.WritePachTokenToConfig(resp.PachToken)
```

Comment:
As discussed earlier, when we enable auth on a test cluster this will also add the auth token to the pach context. This should reduce the risk of a cluster becoming unusable after a failed auth test.

Reply:
:+1: